portfolio/src/main/webapp/index.html

```diff
@@ -1,16 +1,34 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
+
+<head>
+<meta charset="UTF-8">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+
+<body>
+  <div id="fullpage">
+    <!-- TODO(samsog@google.com):Merge Navigation Section -->
+    <div class="section">
       <h1>My Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <p>Click here to get a random message:</p>      
```

Comment:
Please remove the trailing whitespace:
https://google.github.io/styleguide/htmlcssguide.html#Trailing_Whitespace

Reply:
Trailing Whitespace removed in commit 971a8c4